src/services/learninghobbies/index.jsx

```
@@ -249,6 +251,289 @@ export const AddLearningHobbies = ({
   );
 };
 
+export const EditLearningHobbies = ({
+  setEditModal,
```

ESLint

[error] 'setEditModal' is missing in props validation

(react/prop-types)